pages/dkp/kommander/1.3/workspaces/workspace-platform-services/index.md

```diff
@@ -7,54 +7,47 @@ menuWeight: 5
 excerpt: How workspace platform services work
 ---
 
-## Workspace platorm service federation
 
-When attaching a cluster, Kommander federates certain platform services on the newly attached cluster. Operators can use the Kommander UI to customize which platform services to federate to the attached clusters in a given workspace.
+When attaching a cluster, Kommander federates certain platform services on the newly attached cluster. Operators can use the Kommander UI to customize which platform services to deploy to the attached clusters in a given workspace.
 
 On attachment, three factors impact successfully deploying a platform service on the attached cluster:
 
 1. Is the attached cluster a Konvoy cluster or not? For example, a cluster deployed using AWS EKS.
 2. Is the platform service enabled in the Workspace Platform Services settings?
 3. Does the platform service support the version of Kubernetes running on the cluster?
 
-The following tables describe the list of platform services and cluster platform services that are federated on attachment, along with the versions of Kubernetes that they support. If the cluster is running a version of Kubernetes that is not in the supported list, then the platform service will not be federated to that cluster. If the platform service description indicates `only federated on non-Konvoy clusters`, the platform service will not be installed into Konvoy clusters, since it is by default installed with Konvoy via [Kubernetes Base Addons](/dkp/konvoy/1.7/addons/).
+The following tables describe the list of platform services that are federated on attachment, along with the versions of Kubernetes that they support. If the cluster is running a version of Kubernetes that is not in the supported list, then the platform service will not be federated to that cluster. If the platform service description indicates `only federated on non-Konvoy clusters`, the platform service will not be installed into Konvoy clusters, since it is by default installed with Konvoy via [Kubernetes Base Addons](/dkp/konvoy/1.7/addons/).
```

Comment:
```suggestion
The following tables describe the list of platform services that are federated on attachment, along with the versions of Kubernetes that they support. If the cluster is running a version of Kubernetes that is not in the supported list, the platform service is not federated to that cluster. If the platform service description indicates `only federated on non-Konvoy clusters`, the platform service is not installed into Konvoy clusters, since it is by default installed with Konvoy using [Kubernetes Base Addons](/dkp/konvoy/1.7/addons/).
```